src/main/java/com/eva/logic/parser/AddApplicationCommandParser.java

```diff
@@ -0,0 +1,82 @@
+package com.eva.logic.parser;
+
+import static com.eva.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import com.eva.commons.core.index.Index;
+import com.eva.logic.commands.AddApplicationCommand;
+import com.eva.logic.parser.exceptions.ParseException;
+import com.eva.model.person.applicant.application.Application;
+import com.eva.model.person.applicant.application.Education;
+import com.eva.model.person.applicant.application.Experience;
+
+public class AddApplicationCommandParser implements Parser<AddApplicationCommand> {
+
+    /**
+     * Parses {@code userInput} into a command and returns it.
+     * @throws ParseException if {@code userInput} does not conform the expected format
+     */
+    @Override
+    public AddApplicationCommand parse(String userInput) throws ParseException, FileNotFoundException {
+
+        try {
+            String[] substrings = userInput.split(" ");
+            if (substrings.length <= 1) {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                        AddApplicationCommand.MESSAGE_USAGE_2));
+            }
+            String indexNo = substrings[1];
+            String filePath = substrings[2];
+            File file = new File(filePath);
+            Scanner sc = new Scanner(file);
+
+            List<Education> eduList = new ArrayList<>();
+            List<Experience> expList = new ArrayList<>();
+
+            // Name
+            String name = sc.nextLine().split(":")[1];
+            sc.nextLine(); // read blank line
+
+            // Education
+            while (sc.hasNextLine()) {
+                String line = sc.nextLine();
+                // System.out.println(line.split(" ")[0]);
+                if (line.equals("Experience:")) {
+                    break;
+                }
+                if (line.length() > 6 && line.substring(0, 6).equals("School")) {
+                    String schoolName = line.split(" ")[1];
+                    String startDate = sc.nextLine().split(" ")[1];
+                    String endDate = sc.nextLine().split(" ")[1];
+                    Education edu = new Education(startDate, endDate, schoolName);
+                    eduList.add(edu);
+                }
+            }
+
+            // Experience
+            while (sc.hasNextLine()) {
+                sc.nextLine(); // this should be number
+                String company = sc.nextLine().split(" ")[1];
```

Comment:
What if the individual items exceed one line? Does the code handle that

Reply:
Oh right thanks...i will update it to handle job description + catch errors later on.